config/index.js

```diff
@@ -271,6 +271,25 @@ var conf = convict({
       default: ''
     }
   },
+  verificationReminder: {
+    enabled: {
+      doc: 'Is verification reminder enabled',
+      format: Boolean,
+      env: 'VERIFICATION_REMINDER_ENABLED',
+      default: false
+    },
+    queueUrl: {
+      doc: 'The verification reminder queue URL to use (should include https://sqs.<region>.amazonaws.com/<account-id>/<queue-name>)',
+      format: String,
+      env: 'VERIFICATION_REMINDER_QUEUE_URL',
```

Comment:
@dannycoates are queue permissions guarded by QueuePolicy (ref https://github.com/mozilla/fxa-dev/blob/6c922db044531934ed33f09be78222daee9993c5/aws/cloudformation/moz-single.json#L264 ) or something else? How would you test / access the queue locally?